index.js

```diff
@@ -169,14 +169,35 @@ exports.login = async function (options) {
 };
 
 exports.viewAccount = async function (options) {
-    let near = await connect(options);
-    let account = await near.account(options.accountId);
-    let state = await account.state();
-    if (state && state.amount) {
-        state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
+    try {
+        let near = await connect(options);
+        let account = await near.account(options.accountId);
+        let state = await account.state();
+        if (state && state.amount) {
+            state['formattedAmount'] = utils.format.formatNearAmount(state.amount);
+        }
+        console.log(`Account ${options.accountId}`);
+        console.log(inspectResponse.formatResponse(state));
+    } catch(e) {
+        console.log(chalk`\n{bold.red Account {bold.white ${options.accountId}} is not found in {bold.white ${config.helperAccount}} network.\n}`);
```

Comment:
need to check whether error is actually about account being not found (there can be other errors, like e.g. no Internet connection)

Reply:
The same as I described above. 
Is there some doc on server errors response so I don't have to guess what happened.